Makefile.am

```diff
@@ -9,8 +9,8 @@ test:
 	make style unittests integration
 
 #	TODO: Use configure to detect Python versions currently installed
-	make PYTHONVERSION=2.7 -Cpython-api style unittests integration
-	make PYTHONVERSION=3.4 -Cpython-api style unittests integration
+	cd python-api; make PYTHONVERSION=2.7 style unittests integration
+	cd python-api; make PYTHONVERSION=3.4 style unittests integration
```

Comment:
Oops that should be in a different pull request. See the comment on the commit https://github.com/timvideos/gst-switch/commit/bd709218fd59307643c80e728171b0a8b895d166